### ISSUE TITLE
Can't upload picture

### ISSUE THREAD

**Issue by DavidHenryThoreau** [joaovitoriasilva/endurain#8]
```
Warning: Undefined array key 0 in /var/www/html/settings/settings.php on line 266

Warning: move_uploaded_file(../img/users_img/img_19_2941128_9.jpg): Failed to open stream: No such file or directory in /var/www/html/settings/settings.php on line 271

Warning: move_uploaded_file(): Unable to move "/tmp/phpr7Dgf6" to "../img/users_img/img_19_2941128_9.jpg" in /var/www/html/settings/settings.php on line 271

```

That file is a jpeg

```
root@debian-12:~/endurain# file pict.jpg
pict.jpg:                                                     JPEG image data, Exif standard: [TIFF image data, little-endian, direntries=7, orientation=upper-left, xresolution=110, yresolution=118, resolutionunit=2], baseline, precision 8, 1068x1600, components 3

```

**Comment by joaovitoriasilva** [comment]
I believe this is lack of permissions on the Docker image on the folder /var/www/html/.
I did a "chmod 755 -R /var/www/html/" on a clean install on my side and it solved it.

**Comment by DavidHenryThoreau** [comment]
```
root@debian-12:~/endurain# grep -A 1 volume docker-compose.yml
    # Configure volume if you want to edit the code locally by clomming the repo
    volumes:
      - /data/config/endurain/frontend:/var/www/html
--
    # Configure volume if you want to edit the code locally by clomming the repo
    volumes:
      - /data/config/endurain/backend:/app
--
    volumes:
      - /data/config/mariadb:/var/lib/mysql

 chown abeure: /data/config/endurain/{frontend,backend} -R
 docker compose up -d
root@debian-12:~/endurain# docker logs backend  -f
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
```

**Comment by DavidHenryThoreau** [comment]
```
root@debian-12:~/endurain# grep -A 1 volume docker-compose.yml
    # Configure volume if you want to edit the code locally by clomming the repo
    volumes:
      - /data/config/endurain/frontend:/var/www/html
--
    # Configure volume if you want to edit the code locally by clomming the repo
    volumes:
      - /data/config/endurain/backend:/app
--
    volumes:
      - /data/config/mariadb:/var/lib/mysql

 chown rodriguo: /data/config/endurain/{frontend,backend} -R
 docker compose up -d
root@debian-12:~/endurain# docker logs backend  -f
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
ERROR:    Error loading ASGI app. Could not import module "main".
```

**Comment by joaovitoriasilva** [comment]
Fixed on v0.1.4
